layouts/shortcodes/contributors.html

```diff
@@ -0,0 +1,40 @@
+
+<table class="tableblock stretch">
+    <thead>
+        <tr>
+            <th>Avatar</th>
+            <th>Name</th>
+            <th>Organization</th>
+        </tr>
+    </thead>
+    <tbody>
+        {{ range  $page_count := (seq 10) }}
+            {{ $page := string $page_count }}
+                {{ $git_contributors := getJSON "https://api.github.com/repos/apache/camel/contributors?per_page=100&anon=1&page=" $page "&access_token=" $.Site.Params.token }}
+                {{ range  $git_contributors }}
+                    {{ if .url }}
+                        {{ $user_url := getJSON .url }}
+                        {{ if $user_url.name }}
+                            <tr>
+                                <td> <img width="50px" height="50px" src="{{ .avatar_url }}"/></td>
```

Comment:
Would prefer to use a CSS class instead of `width`/`height`

Reply:
fixed